variables.tf

```diff
@@ -12,8 +12,8 @@ variable "infrastructurename" {
 
 variable "linuxNodeSize" {
   type        = list(string)
-  description = "The machine size of the Linux nodes for the regular services"
-  default     = ["m5a.4xlarge", "m5a.8xlarge"]
+  description = "The machine size of the Linux nodes for the regular services, user must check the availability of the instance types for the region"
```

Comment:
. The list is ordered by priority where the first instance type gets the highest priority. Instances types must/should fulfill the following requirements: 64 GB RAM, 16 vCPUs, at least 110 IPs, at least 2 availability zones.

Reply:
Done